judge/models/contest.py

```diff
@@ -474,8 +474,14 @@ class ContestSubmission(models.Model):
                                          help_text=_('Whether this submission was rejudged/rescored '
                                                      'while the contest was frozen.'),
                                          default=False)
+    is_disqualified = models.BooleanField(verbose_name=_('is disqualified'),
+                                     help_text=_('Whether this submission is disqualified.'),
+                                     default=False)
 
     class Meta:
+        permissions = (
+            ('disqualify_submission', _('Disqualify the submission')),
```

Comment:
"Disqualify submissions" is a better persmission name. Please change it to that. Also, update the migration to that as well.